app/views/secondary_content_links/index.html.erb

```diff
@@ -23,13 +23,13 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-  <h2 class="govuk-heading-m">What is secondary content?</h2>
+  <h2 class="govuk-heading-m"><%= t("secondary_content_links.index.header") %></h2>
```

Comment:
This whole section (guidance on secondary links) could be converted to multi-line govspeak too giving a bit more flexibility in how the content is structured.